src/component/mxgraph/shape/render/icon-painter.ts

```diff
@@ -354,6 +354,23 @@ export class IconPainter {
     return canvas;
   }
 
+  /**
+   * Code for creating an asterisk icon used by `complex gateway`.
+   */
+  private createCanvas(paintParameter: PaintParameter): BpmnCanvas {
+    const canvas = this.newBpmnCanvas(paintParameter, { height: 1, width: 1 });
+    return canvas;
+  }
```

Comment:
This method is only used once. Can you remove it and inline the code in `paintAsteriskIcon`?

Reply:
fixed in 19f764662ff882c2a4f02da576e3c3943d4ef0fd and 325c7646ea5de9961e54c16a4fe15888b5b1ab43